spec/lucky/assignable_spec.cr

```diff
@@ -41,11 +43,23 @@ class PageWithQuestionMark
   end
 end
 
+class PageWithDefaultsFirst
+  include Lucky::HTMLPage
+  needs extra_css : String?
```

Comment:
Could you add another needs above this one without a default and not nil? I’m not sure but I think that may show an issue

Reply:
And maybe a default of false too

Reply:
Adding a required one above this and the spec still passes.

Reply:
Ok cool! What about the false default?

Reply:
Cool. I added a few more just for some added comfort. They look good!